src/azure-cli/azure/cli/command_modules/apim/_params.py

```diff
@@ -92,7 +110,28 @@ def load_arguments(self, _):
         c.argument('api_version', help='Indicates the Version identifier of the API if the API is versioned.')
         c.argument('is_current', arg_type=get_three_state_flag(), help='Indicates if API revision is current api revision.')
         c.argument('service_url', help='Absolute URL of the backend service implementing this API. Cannot be more than 2000 characters long.')
-        c.argument('protocols', arg_type=get_enum_type(API_PROTOCOLS), help='Describes on which protocols the operations in this API can be invoked.')
+        c.argument('protocols', arg_type=get_enum_type(API_PROTOCOLS), nargs='+', help='Describes on which protocols the operations in this API can be invoked.')
         c.argument('api_type', arg_type=get_enum_type(API_TYPES), help='The type of the API.')
         c.argument('subscription_required', arg_type=get_three_state_flag(), help='If true, the API requires a subscription key on requests.')
         c.argument('tags', tags_type)
+
+    with self.argument_context('apim api import') as c:
+        c.argument('service_name', options_list=['--service_name', '-n'], help="The name of the api management service instance", id_part=None)
+        c.argument('api_path', help='Required. Relative URL uniquely identifying this API and all of its resource paths within the API Management service instance.', required=True)
+        c.argument('description', help='Description of the API. May include HTML formatting tags.')
+        c.argument('subscription_key_parameter_names', help='Protocols over which API is made available.')
```

Comment:
help message is not consistent with the argument's name for me. Could you give more help message?